actors/miner/src/lib.rs

```diff
@@ -3595,6 +3601,13 @@ pub struct DealsActivationInfo {
     pub deal_ids: Vec<DealID>,
     pub sector_expiry: ChainEpoch,
     pub sector_number: SectorNumber,
+    pub sector_type: RegisteredSealProof,
+}
+
+#[derive(Debug, Clone)]
+pub struct ActivatedSectorInfo {
```

Comment:
Since we're adding a new type here now, I think we can drop use of the imported `market::ext::DealSpaces`. Just inline the two fields into this structure, and also into `UpdateWithDetails`. Then remove the `ext` definition.